src/commands/did-resolve.ts

```diff
@@ -21,7 +21,7 @@ export class DidResolveCommand extends BaseCommand {
         {
             name: "did",
             description:
-                "The identifier to resolve. Expected format : '@[unik:][type,1]/expliciteValue[?propertyKey|?*]'",
+                "The identifier to resolve. Expected format : '@[unik:][type,1]:expliciteValue[?propertyKey|?*]'",
```

Comment:
- I think we can remove '1' as example, user could think that he has to put type or '1' 
- you can move ':' inside type hooks